src/cellarr/dataloader.py

```diff
@@ -1,15 +1,374 @@
+from collections import Counter
+import numpy as np
+import os
+import pytorch_lightning as pl
+from scipy.sparse import coo_matrix, csr_matrix
 import tiledb
+import torch
+from torch.utils.data import Dataset, DataLoader, WeightedRandomSampler
+from typing import List, Optional
+
+from cellarr.queryutils_tiledb_frame import subset_frame
```

Comment:
also remove cellar

```
from .queryutils_tiledb_frame import subset_frame
```

sometimes causes problems with conflicting versions of packages; so we can explicitly ask to look in the relative directory